e2e/utils/pages/homepage.page.ts

```diff
@@ -180,6 +184,20 @@ abstract class Homepage {
 		await this.#page.getByTestId(NAVIGATION_MENU_BUTTON).waitFor();
 	}
 
+	public async navigateToSlide(slideNumber: number): Promise<void> {
+		const navigationTestId = `${CAROUSEL_SLIDE_NAVIGATION}${slideNumber}`;
+		const element = this.#page.locator(`[data-tid="${navigationTestId}"]:visible`);
+		await element.waitFor({ state: 'visible' });
+		await element.click();
+	}
+
+	public async getNumberOfSlides(): Promise<number> {
+		const slides = this.#page.locator(`[data-tid^="${CAROUSEL_SLIDE_NAVIGATION}"]:visible`);
+		await slides.first().waitFor({ state: 'visible' });
+		const count = await slides.count();
+		return count;
+	}
```

Comment:
these two function could go theoretically directly inside the class of the carousel, so that you call them directly from the object

Reply:
done